tools/macos-cleanup.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+# Maintenance script for macOS
+# This script cleans up various caches, temporary files, and other cruft on macOS.
+# Custom script (forked from https://github.com/paulaime/CleanUpMac)
+# -- STEPS ---
+# Download/update the script: curl https://raw.githubusercontent.com/nodejs/build/main/tools/macos-cleanup.sh --output macos-cleanup.sh
+# Make the script executable: chmod +x macos-cleanup.sh
+# Usage: sudo ./macos-cleanup.sh
+
+# Ask for the administrator password upfront
+if [ "$EUID" -ne 0  ]; then
+    echo "Please run as root"
+    exit
+fi
+
+
+echo 'Empty the Trash on all mounted volumes and the main HDD…'
+sudo rm -rfv /Volumes/*/.Trashes &>/dev/null
+sudo rm -rfv ~/.Trash &>/dev/null
+
+echo 'Clear System Log Files…'
+sudo rm -rfv /private/var/log/asl/*.asl &>/dev/null
+sudo rm -rfv /Library/Logs/DiagnosticReports/* &>/dev/null
+sudo rm -rfv /Library/Logs/Adobe/* &>/dev/null
+rm -rfv ~/Library/Containers/com.apple.mail/Data/Library/Logs/Mail/* &>/dev/null
+rm -rfv ~/Library/Logs/CoreSimulator/* &>/dev/null
+
+echo 'Clear Adobe Cache Files…'
+sudo rm -rfv ~/Library/Application\ Support/Adobe/Common/Media\ Cache\ Files/* &>/dev/null
+
+echo 'Cleanup iOS Applications…'
+rm -rfv ~/Music/iTunes/iTunes\ Media/Mobile\ Applications/* &>/dev/null
+
+echo 'Remove iOS Device Backups…'
+rm -rfv ~/Library/Application\ Support/MobileSync/Backup/* &>/dev/null
+
+echo 'Cleanup XCode Derived Data and Archives…'
+rm -rfv ~/Library/Developer/Xcode/DerivedData/* &>/dev/null
+rm -rfv ~/Library/Developer/Xcode/Archives/* &>/dev/null
+
+echo 'Cleanup Homebrew Cache…'
+brew cleanup --force -s &>/dev/null
+brew cask cleanup &>/dev/null
+rm -rfv /Library/Caches/Homebrew/* &>/dev/null
+brew tap --repair &>/dev/null
+
+echo 'Cleanup MacOS Application caches'
+rm -rfv /System/Library/Caches/*
```

Comment:
```suggestion
echo 'Cleanup symbolication cache'
symbolscache delete
```

I found the solution to cleanup without having to reboot afterwards 🎉 

```
administrator@test-orka-macos11-x64-1 ~ % sudo symbolscache status
Target Status
    version:           1
    uptime:            800361 secs
    max entries:       512
    used entries:      344
    bytes total:       14.31 GB

administrator@test-orka-macos11-x64-1 ~ % sudo symbolscache delete
  DELETED: 0074E20F-E7D8-31F5-AF41-0D7BF370CD50     0   system     37732352   16.00 KB      263
  DELETED: 05BB001B-D147-3115-A910-B7FBD66C3772     0   system  11633147904  338.05 MB      337
<truncated>
  DELETED: FFFB49D7-2CA6-3E1F-AE4E-5697B19B7D76     0   system     37470208  160.00 KB      256
  344 entries deleted

administrator@test-orka-macos11-x64-1 ~ % sudo symbolscache status
Target Status
    version:           1
    uptime:            800387 secs
    max entries:       512
    used entries:      0
    bytes total:       32.00 KB

administrator@test-orka-macos11-x64-1 ~ % sudo du -sh /System/Library/Caches
184K	/System/Library/Caches
```

Reply:
wow! This is awesome!